src/explore-education-statistics-admin/src/pages/release/datablocks/components/chart/ChartReferenceLinesConfiguration.tsx

```diff
@@ -134,7 +136,34 @@ export default function ChartReferenceLinesConfiguration({
             }}
             validationSchema={Yup.object<AddFormValues>({
               label: Yup.string().required('Enter label'),
-              position: Yup.string().required('Enter position'),
+              otherAxisPosition: Yup.number().test({
+                name: 'otherAxisPosition',
+                message: 'Enter a valid position',
```

Comment:
Think we could be a little clearer than this as it's not going to be super obvious why this isn't valid. Perhaps something like 'Enter a position within the Y axis min/max range' would be better? If there's something more succinct, happy to use that.

With longer messages, we'll might want to style the 'Add line' button to avoid it squashing like this:

![image](https://user-images.githubusercontent.com/9917868/201092732-1a87cafb-b86b-4e94-9533-5bbc56ef6027.png)

We might want to set a custom width for the 'Actions' column?